src/plugins/formvalid/formvalid-server-en.hbs

```diff
@@ -0,0 +1,127 @@
+<p>To test: Try clearing a field or enter 201 as the Beneficiary's age.</p>
+
+	<h2>
+		<span id="lblTitle" property="name">Demo Asp.Net Web Form</span>
+	</h2>
+
+	<div class="wb-frmvld">
+	<form id="frmMain" action="#" method="get">
+
+		<section tabindex="-1" class="alert alert-danger" id="errors-frmMain">
+			<h2>The form could not be submitted because 2 errors were found.</h2>
+			<ul>
+				<li><a href="#BeneficiaryName"><span class="prefix">Error&nbsp;1: </span>Beneficiary name - The name should begin with "Super" - (Correct and resubmit) </a></li>
+				<li><a href="#BeneficiaryAge"><span class="prefix">Error&nbsp;2: </span>Beneficiary age - Age should be greater than 64 for a Retirement benefit - (Correct and resubmit) </a></li>
+			</ul>
+		</section>
+
+		<noscript>
+			<!-- Fix Error cs0103 -->
+
+			<section class="alert alert-danger wb-server-error-clear" >
+			<h2>This is user control DisplayErrors (It appears when JavaScript is disabled)</h2>
+			<ul>
+			<li><a href="#BeneficiaryName"><span class='prefix'>Error&amp;nbsp;1: </span>Beneficiary name - The name should begin with "Super" - (Correct and resubmit)</a></li>
+			<li><a href="#BeneficiaryAge"><span class='prefix'>Error&amp;nbsp;2: </span>Beneficiary age - Age should be greater than 64 for a Retirement benefit - (Correct and resubmit)</a></li>
+			</ul>
+			</section>
+		</noscript>
+
+		<section class="panel panel-info">
+			<header class="panel-heading">
+				<h2 class="panel-title">
+					Section 1
+				</h2>
+			</header>
+
+			<div class="panel-body">
+				<div class="form-group">
+					<label class="css-star" for="TypeOfPrestation">
+						<abbr title="">
+							<span title="Enter the type of benefit desired" class="field-name" id="TypeOfPrestationLbl">Type of benefit</span>
+						</abbr>
+						<strong class="css-required">
+							(Required field)
+						</strong>
+					</label>
+					<div>
+						<select name="TypeOfPrestation" class="valid" id="TypeOfPrestation" aria-invalid="false" aria-required="true" required="required">
+							<option value="">Select a benefit</option>
+							<option value="ChildrenHelp">Child benefit (Age 1-18) </option>
+							<option value="Invalidity">Invalidity (Age 1-64)</option>
+							<option selected="selected" value="Retirement">Retirement (Age 65-200)</option>
+						</select>
+					</div>
+				</div>
+
+				<fieldset>
+				<legend>Beneficiary</legend>
+
+				<div class="form-group">
+					<label class="css-star" for="BeneficiaryName">
+						<abbr title="">
+							<span title="Enter the beneficiary name" class="field-name" id="BeneficiaryNameLbl">Beneficiary name</span>
+						</abbr>
+						<strong class="css-required">
+							(Required field)
+						</strong>
+						<span class="label label-danger wb-server-error" id="BeneficiaryNameCv" data-fieldname-id="Beneficiary name" data-element-id="BeneficiaryName">
```

Comment:
The ```data-fieldname-id``` seems useless or only needed by the ASP.net implementation.

That information is not used by the plugin. 

For ```data-element-id``` attribute, I think it is not required to add that reference manually. The JS can be updated to add that custom attribute and deduct it's value during the plugin initialisation.

Like (code not optimized and not tested):
```
$( ".wb-server-error" ).each( function() {
    var $parentLabel = $( this ).parentUntil( "label" ).parent();
    $( this ).attr( 'data-element-id', $parentLabel.get(0).id );
} );
```

The ```id``` is not required, I suggest to remove it from the working example

Reply:
data-fieldname-id and data-element-id removed